pkgs/top-level/python-packages.nix

```diff
@@ -2616,17 +2623,29 @@ in modules // {
       py.test $out/${python.sitePackages}
     '';
 
-    # Marked broken since according to test
+    # FIXME: make gdk_pixbuf dependency optional (as wel as xcfffi)
     # Happens with 0.7.1 and 0.7.2
     # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
 
-    patchPhase = ''
+    patches = [
```

Comment:
@globin @abbradar 
Folks, I possible need help/check about inclusion patch from github here  (at least one machine other than host where I develop it refuse to download patch, due hash mismatch)